Research/index.html

```diff
@@ -26,7 +26,7 @@
   {% include card.html
      icon="document"
      title="Key Publications"
-     body="Key papers and articles about seL4 technology, design principles, applications, etc."
+     body="Key papers and articles about the seL4 technology, its design principles and its applications."
```

Comment:
I missed the `the` before. I would have left that out. The `its` is supposed to refer to seL4, not technology. Maybe we should leave out `technology` altogether. E.g.

`Key papers and articles about seL4, its design principles and its applications.`